packages/service-clients/odsp-client/src/odspAudience.ts

```diff
@@ -3,21 +3,42 @@
  * Licensed under the MIT License.
  */
 import { assert } from "@fluidframework/core-utils";
-import { type IClient } from "@fluidframework/protocol-definitions";
+import { IUser, type IClient } from "@fluidframework/protocol-definitions";
 
-import { type OdspMember, OdspUser } from "./interfaces";
+import { type OdspMember } from "./interfaces";
+
+/**
+ * Since ODSP provides user names and oids for all of its members, we extend the
+ * {@link @fluidframework/protocol-definitions#IMember} interface to include this service-specific value.
+ * @internal
+ */
+interface OdspUser extends IUser {
+	/**
+	 * This is the email id of the user
+	 * Overrides the documentation from IUser.id.
```

Comment:
Similar here about not needing to explicitly mention the override, also nit that you can drop the "This is" to be a little less verbose here and below.

```suggestion
	 * The user's email address
```